scripts/run_parachain_ismp_relayer_messaging_integration_tests.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+
+cargo test --package integration-test --lib -- --ignored submit_transfer_function_works -- --exact &&
+cargo test --package integration-test --lib -- --ignored  get_request_works -- --exact
```

Comment:
```suggestion
cargo test --package tesseract-integration-test --lib -- --ignored --nocapture
```

The package should be called tesseract-integration-test, also we don't need this script file. Run the command directly in CI

Reply:
So the issue is running the tests all together one fails due to competing tx priority.
but running them seperately everything works fine, thats why i include the script

Reply:
You can use different accounts for each test

Reply:
Ive resolved this, but is so much cleaner to include the script file and run the tests seperately with 2 commands than just 1 command. The log messages comes so cleaner with 2 commands and you can follow through the individual test execution much better.

But I will do as instructed

Reply:
You can use `test-threads=1` to force cargo to pipeline the test runs